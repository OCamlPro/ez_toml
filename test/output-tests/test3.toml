
w = 1
x = 1
y = 1
z = { a = 1, b = 1 }

# Extensions on operators

w == 2
x := 2
{} -= [ "y" ]
z -= [ "a" ]

