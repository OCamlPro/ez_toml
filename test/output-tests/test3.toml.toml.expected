
w = 1
x = 2

[z]
b = 1

